integration_tests/webhook/test_webhook.py

```diff
@@ -41,12 +36,60 @@ def test_webhook(self):
                     break
             if channel_id is not None:
                 break
+        return channel_id
+
+    def test_webhook(self):
+        url = os.environ[SLACK_SDK_TEST_INCOMING_WEBHOOK_URL]
+        webhook = WebhookClient(url)
+        response = webhook.send(text="Hello!")
+        self.assertEqual(200, response.status_code)
+        self.assertEqual("ok", response.body)
 
-        history = client.conversations_history(channel=channel_id, limit=1)
+        token = os.environ[SLACK_SDK_TEST_BOT_TOKEN]
+        client = WebClient(token=token)
+        history = client.conversations_history(channel=self.__get_channel_id(), limit=1)
         self.assertIsNotNone(history)
         actual_text = history["messages"][0]["text"]
         self.assertEqual("Hello!", actual_text)
 
+    def test_with_unfurls_off(self):
+        url = os.environ[SLACK_SDK_TEST_INCOMING_WEBHOOK_URL]
+        token = os.environ[SLACK_SDK_TEST_BOT_TOKEN]
+        webhook = WebhookClient(url)
+        client = WebClient(token=token)
+        # send message that does not unfurl
+        response = webhook.send(
+            text="<https://imgs.xkcd.com/comics/desert_golfing_2x.png|Desert Golfing>",
+            unfurl_links=False,
+            unfurl_media=False,
+        )
+        self.assertEqual(200, response.status_code)
+        self.assertEqual("ok", response.body)
+        # wait in order to allow Slack API to edit message with attachments
+        time.sleep(2)
+        history = client.conversations_history(channel=self.__get_channel_id(), limit=1)
+        self.assertIsNotNone(history)
+        self.assertTrue("attachments" not in history["messages"][0])
+
+    def test_with_unfurls_on(self):
+        url = os.environ[SLACK_SDK_TEST_INCOMING_WEBHOOK_URL]
+        token = os.environ[SLACK_SDK_TEST_BOT_TOKEN]
+        webhook = WebhookClient(url)
+        client = WebClient(token=token)
+        # send message that does unfurl
+        response = webhook.send(
+            text="<https://imgs.xkcd.com/comics/red_spiders_small.jpg|Spiders>",
+            unfurl_links=True,
+            unfurl_media=True,
+        )
+        self.assertEqual(200, response.status_code)
+        self.assertEqual("ok", response.body)
+        # wait in order to allow Slack API to edit message with attachments
+        time.sleep(2)
```

Comment:
For review: Inserted a sleep here because first the message (no unfurl is posted), then it is edited by Slack API with the unfurl attachments. To avoid race condition, we wait an arbitrary long period, then make the conversations.history call to fetch the edited message.